.circleci/config.yml

```diff
@@ -2,33 +2,35 @@ version: 2
 jobs:
   build:
     docker:
-      - image: docker:18-git
+      - image: docker:17.03-git
     environment:
-      APP_NAME: "get-access/cla_public"
+      DOCKER_REGISTRY: "registry.service.dsd.io"
+      DOCKER_IMAGE: "cla_public"
     steps:
       - checkout
       - setup_remote_docker:
+          version: 17.03.0-ce
```

Comment:
Why downgrade Docker version?

Reply:
This is the latest known version which the registry works with. Once this is merged, we could explore upgrading it one by one.